external/js/cothority/src/byzcoin/config.ts

```diff
@@ -46,6 +46,19 @@ export default class ChainConfig extends Message<ChainConfig> {
             },
         });
     }
+
+    /**
+     * Return a string representation of a config
+     */
+    toString(): string {
```

Comment:
Couldn't that use directly the new pretty-printing?

Reply:
I would rather not. pretty-printing is about printing a transaction, toString() is about printing an instance.

Reply:
I suppose you'll need to print instances, too. At least that's what I'm doing in my `ol-explorer`, and it's handy - I had to undo a deletion, again ;)

Reply:
Sure, something we can still add